Tests/Unit/Extend/Controller/OrderControllerTest.php

```diff
@@ -0,0 +1,180 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+use OxidEsales\Eshop\Application\Model\User;
+use OxidEsales\Eshop\Core\DatabaseProvider;
+use OxidEsales\Eshop\Core\Field;
+use OxidEsales\Eshop\Core\Price;
+use OxidEsales\Eshop\Core\Registry;
+
+use Wirecard\Oxid\Extend\Controller\OrderController;
+use Wirecard\Oxid\Extend\Model\Basket;
+
+class OrderControllerTest extends \Wirecard\Test\WdUnitTestCase
+{
+    /**
+     * @var OrderController
+     */
+    private $_controller;
+
+    protected function setUp()
+    {
+        $this->_controller = oxNew(OrderController::class);
+
+        parent::setUp();
+    }
+
+    public function testInit()
+    {
+
```

Comment:
remove the newline, maybe name the test after when it happens that you get returned true?